debug/package.json

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
Update version of the module in the package and you can hopefully test a hot rollout!